File: Java/Q45.java
/*
 Q45- Program to Calculate Standard Deviation
 */



import java.util.Scanner;

public class Q45{
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter number of elements: ");
        int n=sc.nextInt();

        double[] arr=new double[n];
        double sum=0.0;

        System.out.println("Enter elements:");
        for(int i=0; i<n; i++){
            arr[i]=sc.nextDouble();
            sum += arr[i];
        }

        double mean = sum/n;
        double variance = 0.0;

        for(double num : arr){
            variance+=Math.pow(num-mean, 2);
        }

        variance/=n;
        double stdDev = Math.sqrt(variance);

        System.out.println("Standard Deviation = " + stdDev);
        sc.close();
    }
}

File: C++/Q45.cpp
// Q45.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main() {
    int n;
    cout << "Enter number of elements: ";
    cin >> n;
    double arr[n], sum = 0, mean, sd = 0;
    for (int i = 0; i < n; i++) {
        cin >> arr[i];
        sum += arr[i];
    }
    mean = sum / n;
    for (int i = 0; i < n; i++)
        sd += pow(arr[i] - mean, 2);
    cout << "Standard Deviation = " << sqrt(sd / n);
    return 0;
}
